### DISCUSSION TITLE
Select "multiple" options in latest version not working

### DISCUSSION THREAD

**Discussion by nishkohli96** [react-hook-form/react-hook-form#5776] [Q&A] [answered]
I recently upgraded **react-hook-form** from `6.14.1` to `7.9.0`(latest) in my project. I have made the relevant changes in code, which works fine as per the latest version. However, when using the `multiple` prop with `MUI-Select`, the app crashed, and it also selects only one value when using the native `Select `.

Attaching the link to the sandbox [here](https://codesandbox.io/s/romantic-feynman-0hl5i?file=/README.md) with the problem description in README.md and code in App.js file.

**Comment by bluebill1049** [comment, marked as answer]
can you share more details? like how to reproduce the problem and etc.

**Reply by nishkohli96** [reply]
Attached in the sandbox [here](https://codesandbox.io/s/romantic-feynman-0hl5i?file=/README.md). All I did was add the `multiple` property to the `Select` and bangg, it crashed.  Coudn't find any solution on the web. 

Tried to manipulate it like 

 ```
<Controller
      name="mentor"
      type="select"
      control={control}
      defaultValue={initialValues.mentor}
      rules={{ required: true }}
      render={({ field }) => (
          <Select
                value={[...field?.value]}
                multiple
           >             
           // select opts
        </Select>
        )}
 />
```
, just like I did that in version `6.14.1` [here](https://stackoverflow.com/a/65780275), but it failed.

**Reply by bluebill1049** [reply]
<img width="969" alt="Screen Shot 2021-06-29 at 8 02 45 pm" src="https://user-images.githubusercontent.com/10513364/123778933-fb159100-d914-11eb-9ccc-23c439a059db.png">

did you supply a `defaultValues` or defaultValue with array-like for the value? that's what the error said.

**Reply by nishkohli96** [reply]
`defaultValue={[animals[0]]}` , yeah this fixed the issue, atleast for the MUI select. Thanks for the help Bill

**Reply by bluebill1049** [reply]
np 👍
